llvmlite/binding/passmanagers.py

```diff
@@ -53,6 +53,34 @@ def dump_refprune_stats(printout=False):
                       stats.fanout_raise)
 
 
+def set_time_passes(enable):
+    """Enable or disable pass timers.
+
+    Parameters
+    ----------
+    enable : bool
+        Set to True to enable pass timers.
+        Set to False to disable pass timers.
+    """
+    ffi.lib.LLVMPY_SetTimePasses(c_int(enable))
```

Comment:
Can this be a `c_bool` and then the binding also use bool?

Reply:
fixed in https://github.com/numba/llvmlite/pull/624/commits/bf2159ea486292245c0ca99d87a5bd70c55159f2